notebooks/Airline Delays - Data Processing Pipeline.py

```diff
@@ -658,6 +677,38 @@ def chain_delay_analysis (dep_time_diff_one_flight_before, dep_time_diff_two_fli
 
 # COMMAND ----------
 
+# Index label
+labelIndexer = StringIndexer(inputCol="dep_del15", outputCol="label")
+
```

Comment:
The indexer should only be fit on the train set.
```suggestion
labelIndexer.fit(train_GBT)
```